File: 1035.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		Scanner t = new Scanner(System.in);
		int A = t.nextInt();
		int B = t.nextInt();
		int C = t.nextInt();
		int D = t.nextInt();

		if (B > C && D > A && (C + D) > (A + B) && C > 0 && D > 0 && A % 2 == 0)
			System.out.printf("Valores aceitos\n");
		else
			System.out.printf("Valores nao aceitos\n");
	}
}

File: 1035.py
# -*- coding: utf-8 -*-

entrada = raw_input().split(" ")
numeros = [int(numero) for numero in entrada] 

A,B,C,D = numeros
if B>C:
    if D>A:
        if (C+D)>(A+B):
            if C>0:
                if D>0:
                    if A%2==0:
                        print "Valores aceitos"
                    
                    else:
                        print "Valores nao aceitos"
                else:
                    print "Valores nao aceitos"    
            else:
                print "Valores nao aceitos"
        else:
            print "Valores nao aceitos"
    else:
        print "Valores nao aceitos"
else:
    print "Valores nao aceitos" 